### DISCUSSION TITLE
How to enable syntax highlighting for other languages.

### DISCUSSION THREAD

**Discussion by wylited** [nyoom-engineering/nyoom.nvim#76] [Q&A] [answered]
How do I enable syntax highlighting for other languages like markdown or org-mode.
(I don't know how to.)

**Comment by shaunsingh** [comment, marked as answer]
Uncommenting a module in `modules.fnl`, then run `nyoom sync`. Next time you open neovim it will automatically install the tree sitter parser (and enable syntax highlighting) for the language. 

Keep in mind org-mode support isn't fully done yet, so it might be a bit buggy

**Comment by wylited** [comment]
thanks, i notice that its a bit hard for me to figure out how to do things with nyoom. Would it be possible for me to help with begineer documentation on github wiki or an external website? (I would be more than happy to help where I can.)
